lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -243,12 +240,14 @@ def close
     rescue StandardError => e
       publish_exception('Error when removing remote working dir', e, false)
     ensure
-      @session.close if @session && !@session.closed?
+      system("/usr/bin/ssh", @host, "-o", "User=#{@ssh_user}", "-o", "ControlPath=#{@local_working_dir}/ssh-%r@%h", "-O", "exit")
```

Comment:
This is also used in the polling runner, could we extract it into a common method?